src/main/resources/openapi/schemas/uploadRequest.json

```diff
@@ -13,8 +13,7 @@
          "description": "Key within scope for this setting"
        },
        "value": {
-         "type": "object",
```

Comment:
I love that just deleting this line makes things work so much more flexibly. I reminds me of how you can write type-independent code in languages like Ruby by ... just not talking about types.